src/common/pf_dcp.c

```diff
@@ -492,8 +492,13 @@ static int pf_dcp_get_req (
          value_length = (uint16_t)strlen ((char *)p_value);
          break;
       case PF_DCP_SUB_DEV_PROP_ALIAS:
-         skip = append_alias_name ? 0 : 1;
-         value_length = (uint16_t)strlen ((char *)p_value);
+         skip = alias_name ? 0 : 1;
+         if (skip == 0)
```

Comment:
Skip is a boolean, `skip = alias_name == NULL` and `if (!skip)`?

Reply:
Done